src/commentsBody/commentsBody.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './commentsBody.css'
+const Comment = React.lazy(() => import('@/comments/comments'));
+const AddMessage = React.lazy(() => import('@/addMessage/addMessage'));
+const CommentsBody = () => {
+    return (
+        <div className='comments_container'>
+            <React.Suspense fallback={<div>Loading...</div>}>
```

Comment:
why do you need suspense here? are there any async operations happening in child components?